mmv1/third_party/terraform/services/cloudsecuritycompliance/resource_cloud_security_compliance_framework_test.go

```diff
@@ -0,0 +1,121 @@
+package cloudsecuritycompliance_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccCloudSecurityComplianceFramework_basic(t *testing.T) {
```

Comment:
Does this add value compared to the [generated test based on the basic example](https://github.com/modular-magician/terraform-provider-google-beta/compare/713b6975566aab2b4d10691efcee1bc880c25655..aa44784d861142e968ab06bf51882a0a153b8c78#diff-1d6263410e3c8aecf40e5dfff1e72880dac75e5eb14a31a020599b1f5319cb8aR53)?